README.md

```diff
@@ -298,6 +298,65 @@ pip install -r requirements.txt
 You can then build the documentation by running ``make <format>`` from the
 ``docs/`` folder. Run ``make`` to get a list of all available output formats.
 
+### Building the CPU-version Libtorch for 32-bit on Windows
+Please use Visual Studio 2017 to the latest version along with installing "VC++ 2017 version 15.4 v14.11 toolset".
+Please use [Anaconda](https://www.anaconda.com/distribution/#download-section) to set 32-bit environment by yourself.
+You need install Python in 32-bit,recommend the 3.7 version.
+Most of steps are same as build pytorch on Windows, a little different with building Libtorch for 32-bit on Windows:
+
+```cmd
+Do all steps in Anaconda Prompt
+:: [Optional]If you have not got the pytorch source, you can git clone pytorch repository here.
+git clone --recursive https://github.com/pytorch/pytorch
+cd pytorch
+git submodule sync
+git submodule update --init --recursive
+
+:: [Optional]Only add the next line if you want to generate Libtorch's libraries in debug mode. If you want libraries in release mode, ignore the next line.
+set DEBUG=1
+
+:: Switch anaconda's platform from win-64 to win-32.
+set CONDA_FORCE_32BIT=1
+
+:: Recommand to create a new enviroment in 32-bit, and activate this environment to do all next steps.
+:: Make sure in your environment,Python is in 32 bit
+:: If Python shows information like this [MSC v.1912 32 bit (Intel)],which mean its architecture is 32-bit. 
+conda create -n env_name python=3.7
+conda activate env_name
+
+:: Install common Dependencies.
+conda install numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing
+
+:: Disable CUDA, for just build CPU-version Libtorch
+set USE_CUDA=0
+
+:: Disable NINJA, and set CMake generate to Visual Studio 15 2017.
+set USE_NINJA=OFF
```

Comment:
Is there a specific reason that ninja is disabled?

Reply:
At the early stage of testing, NINJA made a series of failure which seem to conflict with  Visual Studio 15 2017 CMake generator. So I disable NINJA since that, it never bother me anymore.